kformat/exception.py

```diff
@@ -0,0 +1,30 @@
+from typing import Type
+
+__all__ = ['KFormatError', 'WrongTypeError', 'TooLongValueError']
+
+
+def _name(type_: Type) -> str:
+    return type_.__name__ if hasattr(type_, '__name__') else str(type_)
+
+
+class KFormatError(Exception):
+    pass
+
+
+class WrongTypeError(KFormatError):
+    def __init__(self, expect: Type, value):
+        super().__init__()
+        self.expect = expect
+        self.value = value
+
+    def __str__(self) -> str:
+        return f'Should be "{_name(self.expect)}" instead of "{_name(self.value)}"'
+
+
+class TooLongValueError(KFormatError):
+    def __init__(self, length: int):
+        super().__init__()
+        self.length = length
+
+    def __str__(self) -> str:
+        return f'Too long value is given(max: {self.length})'
```

Comment:
```suggestion
        return f'Invalid length of value is given(expected: {self.length})'
```

그리고 만약 위 오류명 변경 제안에 동의하신다면 위와 같이 메시지를 바꾸면 어떨까 합니다.